package.json

```diff
@@ -331,7 +338,13 @@
     "chromatic": "chromatic --exit-zero-on-changes",
     "change:add": "changeset add",
     "release:prepare-main": "./scripts/prepare-main-after-release.sh",
-    "format:code": "prettier --ignore-unknown --write src/*"
+    "format:code": "prettier --ignore-unknown --write src/*",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/mochareports/report.json",
+    "generate:report": "marge cypress/reports/mochareports/report.json --reportDir cypress/reports/mochareports",
+    "html:report": "npm run combine:reports && npm run generate:report",
+    "create:artifacts:dirs": "mkdir -p cypress/reports/mochareports && mkdir -p cypress/reports/mochareports/videos",
+    "move:screenshots": "find cypress/reports/mochareports -type d -name \"*.js\" -exec mv {} cypress/reports/mochareports \\;",
+    "move:videos": "find cypress/videos -type f -name \"*.js.mp4\" -exec mv {} cypress/reports/mochareports/videos \\;"
```

Comment:
can we combine al of the QA acions under one namespace? eg `qa:XX` -> `qa:move-videos`, `qa:create-adrifacts` etc